connectors/src/connectors/slack/temporal/activities.ts

```diff
@@ -1,4 +1,8 @@
-import { CoreAPIDataSourceDocumentSection, ModelId } from "@dust-tt/types";
+import {
+  CoreAPIDataSourceDocumentSection,
+  memoize,
```

Comment:
the package naming is already outdated 😓

Reply:
@fontanierh we can change it anytime, no big deal.